### PR TITLE
feat(console): page settings

### PR DESCRIPTION
**Pull request by wangsijie**
## Summary

Add page: settings.

## Linear Issue Reference

LOG-2116

## Testing

截屏2022-04-22 下午3 38 49

### CODE REVIEW

File: packages/console/src/pages/Settings/index.tsx [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+          <FormField isRequired title="admin_console.settings.appearance">
+            <Controller
+              name="adminConsole.appearanceMode"
+              control={control}
+              render={({ field: { value, onChange } }) => (
+                <Select
+                  value={value}
+                  options={[
+                    {
+                      value: AppearanceMode.SyncWithSystem,
```
Comment (demonzoo) [inline packages/console/src/pages/Settings/index.tsx:97]: nit: FollowSystem?

File: packages/console/src/pages/Settings/index.tsx [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+                      title: t('settings.appearance_system'),
+                    },
+                    {
+                      value: AppearanceMode.LightMode,
```
Comment (demonzoo) [inline packages/console/src/pages/Settings/index.tsx:101]: nit: just `Appearance.Light/LightMode` would be fine?

Comment (wangsijie) [reply packages/console/src/pages/Settings/index.tsx:101]: I am not sure if there will be other "appearance" settings in the future, this word is more then just dark/light mode.

File: packages/console/src/pages/Settings/index.tsx (hunk 1 above)
Comment (wangsijie) [reply packages/console/src/pages/Settings/index.tsx:97]: I copied this from other websites.

**Review by simeng-li — APPROVED** [review]
LGTM

**Comment by wangsijie** [comment]
Resolve @demonzoo 's conversations since they are "nit"

**Pull request merged**